src/modules/add_remove_task.js

```diff
@@ -0,0 +1,45 @@
+import { fromEvent, debounceTime } from 'rxjs';
+import Todo from './todo.model.js';
+
+// eslint-disable-next-line import/no-mutable-exports
```

Comment:
- [ ] I would advise you to fix this linter manually as said in the previous review. You can use [`StackOverflow`](https://stackoverflow.com/questions/67713204/eslint-disable-default-export) website for research on how to fix the challenge you are facing.